CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.26.0 Release
+- Release Date: December 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### ID-loaded push notifications (E20)
+
+#### Upgraded RN to v0.61
+
+### Improvements
+- Added support for pasting other file types such as videos, PDFs, and Google Docs.
+- Added the option to convert public channels to private in the channel info screen.
+- Made usernames in system messages tappable.
+- Added a count for pinned posts header icon.
+- Added an autocomplete to edit post screen.
+- Updated the channel name length character limit to 64 to match server.
+- Added an expand button to truncated markdown tables to improve discoverability of opening them in full screen.
+- Added an error message when trying to share too long text from share extension.
+- Improved behaviour where posts from different authors in the same thread appeared to be from different threads if separated by new message line.
+- Added support for native emojis in the emoji picker and autocomplete.
+- Removed "Add Reaction" option from the long post view.
+- Improved behaviour of large number of emojis when a post changed from horizontal scroll to wrap.
+- Added support for a generic error message in interactive dialog responses.
+- Added the ability to disable attachment buttons and fields.
+
+### Bug Fixes
+- Fixed an issue on Android where the app slowed down when opening a channel with large number of animated emoji.
+- Fixed an issue where the app crashed when pasting a large file to the text box from the clipboard.
+- Fixed an issue where the app crashed when previewing an image attachment.
+- Fixed an issue where channels archived via the command line interface were still visible on the left-hand side and accessible on mobile apps.
+- Fixed an issue where the thread header in landscape view was wider than the main channel view header.
+- Fixed an issue where sidebar separator line was misaligned between Teams and Channel view.
+- Fixed an issue on iOS where the channel spinner appeared black on a dark theme.
+- Fixed an issue where an asterisk appeared on the "Nickname" and "Position" fields in Edit Profile screen even though nickname is not handled through the login provider.
+- Fixed an issue where the filtered list for emojis opened above the edit box and behind the channel header when adding an emoji to channel header using ``:emoji:``.
+
```

Comment:
Issue when the app was not able to reproduce YouTube videos

Reply:
Super important fix an issue where the app was not blurring the screen on Android when the EMM setting was applied

Reply:
Fix an issue where images/videos could not be saved

Reply:
Fixed a crashed cause by using the emoji category selector

Reply:
"Super important fix an issue where the app was not blurring the screen on Android when the EMM setting was applied"

This was already fixed in v1.25.1 https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md#1251-release.